docs/logo.jl

```diff
@@ -0,0 +1,47 @@
+function generate_logo()
+    # Load original image (from https://github.com/JuliaImages/juliaimages.github.io/blob/source/docs/src/assets/logo.png)
+    img = load("docs/logo_original.png")
+
+    # Define some colors
+    c_red, c_green, c_blue, c_purple = RGBA.([Colors.JULIA_LOGO_COLORS...])
+    gray_dark = RGBA{N0f8}(0.1,0.1,0.1)
+    gray_light = RGBA{N0f8}(0.9,0.9,0.9)
+
+    # Change base color black to gray
+    img = img*N0f8(0.5) .+ RGBA{N0f8}(0.5,0.5,0.5,0)
+    img_dark = copy(img)
+    img_light = copy(img)
+
+    # Define points on the image
+    p_green = Point(87,35)
+    p_red = Point(75,55)
+    p_purple = Point(99,55)
+    p_1 = Point(13,96)
+    p_2 = Point(52,50)
+    p_3 = Point(73,73)
+    p_4 = Point(84,62)
+    p_5 = Point(116,97)
+
+    # Define path
+    path_dark = "docs/src/assets/logo-dark.png"
+    path_light = "docs/src/assets/logo.png"
```

Comment:
The `docs/make.jl` file might be called from two typical paths:

- `project_dir`: `include("docs/make.jl")`
- `project_dir/docs`: `include("make.jl")`

The current version only handles the first case. To also support the second case, might be good to use `@__DIR__` macro here, which will be automatically expanded to `@__DIR__() = "project_dir/docs"`

Also use `joinpath` for windows users.

```suggestion
    path_dark = joinpath(@__DIR__, "src", "assets", "logo-dark.png")
    path_light = joinpath(@__DIR__, "src", "assets", "logo.png")
```

Reply:
Thanks for the suggestion!